### PR TITLE
Feature/separate wix crawler

### PR DESCRIPTION
**Pull request by MikoWhy91**
Right now we have two hits to the API in case of Zielone Tarasy.
We need to change the way how all parsers are handled.

### CODE REVIEW

File: app/services/parsers/ztParser.js [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-const ztService = require('../zieloneTarasyService')
+const wixCrawler = require('../wixCrawler')
 
 const ztParser = $ => {
-  const page = $.html()
-  return ztService.getZieloneTarasyPageObj(page)
-    .then(pageObj => {
-      const menuObj = JSON.parse(pageObj)
-      const menuHTML = menuObj['data']['document_data']['c2pd']['text']
+  return wixCrawler.crawlWix('https://www.zielone-tarasy.eu/', 'aktualności').then(pageObj => {
```
Comment (tpater) [inline app/services/parsers/ztParser.js]: I prefer .then in new line, but that's just a preference

File: app/services/wixCrawler.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+const co = require('co')
+
+/***
+ * Returns crawled page object from wix base sites
+ *
+ * @param uri
+ * @param pageTitle
+ * @returns {Promise}
+ */
+const crawlWix = (uri, pageTitle) => {
```
Comment (tpater) [inline app/services/wixCrawler.js]: wixCrawler.crawlWix - sounds a bit redundant. Let's stick with consistent interface wixCrawler.crawl

File: package.json [hunk 3]
```diff
@@ -35,6 +36,7 @@
     "rimraf": "^2.6.1",
     "standard": "^10.0.2",
     "tap-spec": "^4.1.1",
-    "tape": "^4.6.3"
+    "tape": "^4.6.3",
+    "uglify-js": "^3.0.20"
```
Comment (tpater) [inline package.json]: Where do we use that?

File: app/helpers/helpers.js [hunk 4]
```diff
@@ -1,17 +0,0 @@
-const getPublicModelObj = (data) => {
```
Comment (tpater) [inline app/helpers/helpers.js:1]: Yeah. Got rid of it!

File: package.json (hunk 3 above)
Comment (MikoWhy91) [reply package.json]: It was added by mistake

**Review by tpater — APPROVED** [review]

**Pull request merged**